cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -901,14 +917,56 @@ abstract class TranslatedElement extends TTranslatedElement {
    * Gets the successor instruction of the instruction that was generated by
    * this element for tag `tag`. The successor edge kind is specified by `kind`.
    */
-  abstract Instruction getInstructionSuccessor(InstructionTag tag, EdgeKind kind);
+  abstract Instruction getInstructionSuccessorInternal(InstructionTag tag, EdgeKind kind);
```

Comment:
We should probably update the QLDoc to mention why the predicate is called `Internal` now? And the same goes for `getChildSuccessorInternal`